src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceService.java

```diff
@@ -405,6 +409,16 @@ public void grantPoliciesBqJobUser(String dataProject, Collection<String> policy
         enableIamPermissions(policyMap, dataProject);
     }
 
+    public void revokePoliciesBqJobUser(String dataProject, Collection<String> policyEmails)
+        throws InterruptedException {
+
+        Map<String, List<String>> policyMap = new HashMap<>();
+        List<String> emails = new ArrayList<>(CollectionUtils.collect(policyEmails, e -> "group:" + e));
+        policyMap.put(BQ_JOB_USER_ROLE, emails);
```

Comment:
If there's only one element in a map you can use `singletonMap`
```suggestion
        policyMap = Collections.singletonMap(BQ_JOB_USER_ROLE, emails);
```

Reply:
👍